Source/WTF/wtf/Platform.h

```diff
@@ -82,20 +82,16 @@
    belong as part of Platform.h at all. */
 
 
-#if PLATFORM(GTK)
+#if USE(GLIB)
 #define GLIB_VERSION_MIN_REQUIRED GLIB_VERSION_2_56
 #define GLIB_VERSION_MAX_ALLOWED GLIB_VERSION_2_70
+#endif
+
 #if USE(GTK4)
 #define GDK_VERSION_MIN_REQUIRED GDK_VERSION_4_0
 #else
 #define GDK_VERSION_MIN_REQUIRED GDK_VERSION_3_22
```

Comment:
Now we are defining this for other ports, we still need a `#if PLATFORM(GTK)`

Reply:
thanks for spotting this!